3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/CampaignAttributionEnrichment.scala

```diff
@@ -106,16 +115,18 @@ case class MarketingCampaign(
  * @param mktTerm List of marketing term parameters
  * @param mktContent List of marketing content parameters
  * @param mktCampaign List of marketing campaign parameters
+ * @param mktClick: Map of click ID parameters to networks
  */
 case class CampaignAttributionEnrichment(
   mktMedium:   List[String],
   mktSource:   List[String],
   mktTerm:     List[String],
   mktContent:  List[String],
-  mktCampaign: List[String]
+  mktCampaign: List[String],
+  mktClickId:  List[(String, String)]
   ) extends Enrichment {
```

Comment:
Should we also update the field names in the configuration JSON? https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/campaign_attribution/jsonschema/1-0-1